variables.tf

```diff
@@ -227,6 +227,46 @@ variable "jira_db_master_password" {
   }
 }
 
+variable "jira_shared_home_size" {
+  description = "Storage size for Jira shared home"
+  type        = string
+  default     = "10Gi"
```

Comment:
Again, seems we have defaults on all 3 levels. Is that something we want? Would it be enough to have just 1 level with defaults (e.g. the top one)?